### PR TITLE
TryFrom impls for augrim message to scabbard types

### PR DESCRIPTION
**Pull request by peterschwarz**

### CODE REVIEW

File: services/scabbard/libscabbard/src/service/v3/consensus/process.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use splinter::service::ServiceId;
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct ScabbardProcess(ServiceId);
+
+impl augrim::Process for ScabbardProcess {}
```
Comment (vaporos) [inline services/scabbard/libscabbard/src/service/v3/consensus/process.rs:20]: Add a use augrim::Process above

File: services/scabbard/libscabbard/src/service/v3/consensus/value.rs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use augrim::Value;
+
+#[derive(Clone)]
+pub struct ScabbardValue(Vec<u8>);
```
[the remark this answers is not in the document]
Comment (peterschwarz) [reply services/scabbard/libscabbard/src/service/v3/consensus/value.rs:18]: I believe this is a temporary value as we continue work with the other components, such as the publisher, etc

File: services/scabbard/libscabbard/src/service/v3/consensus/process.rs (hunk 1 above)
Comment (peterschwarz) [reply services/scabbard/libscabbard/src/service/v3/consensus/process.rs]: Fixed

**Comment by peterschwarz** [comment]
Moved the implementation of the try from to the appropriate store submodule. As this has been rebased on main which now includes the Consensus Action Runner, I've also added a feature to cover the consensus module more broadly, and includes the action runner-specific feature under it.

**Review by agunde406 — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Pull request merged**